_migrations/getting-started-data-migration.md

```diff
@@ -0,0 +1,331 @@
+
+    ```bash
+    aws ssm start-session --document-name BootstrapShellDoc-<stage>-<aws-region> --target <instance-id> --region <aws-region> [--profile <profile-name>]
+    ```
+    
+3. Once logged in, run the following command from the shell of the bootstrap instance inside the `/opensearch-migrations` directory):
+
+    ```bash
+    ./initBootstrap.sh && cd deployment/cdk/opensearch-service-migration
+    ```
+    
+4. After a successful build, remember the path for infrastructure deployment in the next step.
+
+---
+
+## Step 4: Configuring and deploying for RFS (~20 mins)
+
+Use the following step to configure and deploy RFS:
+
+1. Add the target cluster password to AWS Secrets Manager as an unstructured string. Be sure to copy the secret ARN for use during deployment.
+2. From the same shell as the bootstrap instance, modify the `cdk.context.json` file located in the `/opensearch-migrations/deployment/cdk/opensearch-service-migration` directory:
+
+    ```json
+    {
+    "migration-assistant": {
+        "vpcId": "<TARGET CLUSTER VPC ID>",
+        "targetCluster": {
+            "endpoint": "<TARGET CLUSTER ENDPOINT>",
+            "auth": {
+                "type": "basic",
+                "username": "<TARGET CLUSTER USERNAME>",
+                "passwordFromSecretArn": "<TARGET CLUSTER PASSWORD SECRET>"
+            }
+        },
+        "sourceCluster": {
+            "endpoint": "<SOURCE CLUSTER ENDPOINT>",
+            "auth": {
+                "type": "basic",
+                "username": "<TARGET CLUSTER USERNAME>",
+                "passwordFromSecretArn": "<TARGET CLUSTER PASSWORD SECRET>"
+            }
+        },
+        "reindexFromSnapshotExtraArgs": "<RFS PARAMETERS (see below)>",
+        "stage": "dev",
+        "otelCollectorEnabled": true,
+        "migrationConsoleServiceEnabled": true,
+        "reindexFromSnapshotServiceEnabled": true,
+        "migrationAssistanceEnabled": true
+    }
+    }
+    ```
+
+    The source and target cluster authorization can be configured to have no authorization, `basic` with a username and password, or `sigv4`. 
+
+3. Bootstrap the account with the following command:
+
+    ```bash
+    cdk bootstrap --c contextId=migration-assistant --require-approval never 
+    ```
+
+4. Deploy the stacks:
+
+    ```bash
+    cdk deploy "*" --c contextId=migration-assistant --require-approval never --concurrency 5
+    ```
+
+5. Verify that all CloudFormation stacks were installed successfully.
+
+### RFS parameters
+
+If you're creating a snapshot using migration tooling, these parameters are auto-configured. If you're using an existing snapshot, modify the  `reindexFromSnapshotExtraArgs` setting with the following values:
+
+    ```bash
+    --s3-repo-uri s3://<bucket-name>/<repo> --s3-region <region> --snapshot-name <name>
+    ```
+
+You will also need to give access to the `migrationconsole` and `reindexFromSnapshot` taskRole permissions to the EC2 bucket. 
+
```

Comment:
Line 162: I assume we mean "S3 bucket". Otherwise, "EC2 instance".

Reply:
Should be S3 bucket.